modules/sdk/project-templates/project-templates/src/main/java/com/liferay/project/templates/internal/util/FileUtil.java

```diff
@@ -125,74 +132,43 @@ public FileVisitResult preVisitDirectory(
 	}
 
 	public static void extractDirectory(
-			String dirName, final Path destinationDirPath)
+			String dirName, final Path destinationDirectoryPath)
```

Comment:
For consistency, please rename `destinationDirectoryPath` to `destinationDirPath`